source/guide_bitwarden.rst

```diff
@@ -0,0 +1,303 @@
+ [isabell@stardust bitwarden_rs]$ tar -xvzf bw_web_v2.13.2.tar.gz
+ [isabell@stardust bitwarden_rs]$ rm -r bw_web_v*.tar.gz
+
+ 
+Generate an ``openssl -base64`` key now and save it temporarily, you'll need it in the next step.
+
+.. code-block:: console
+
+ [isabell@stardust bitwarden_rs]$  openssl rand -base64 48
+ Ig/8bZXhqVFK11F2tQZTfODO/6QpCHE3DGyCH/2Eh40xUWMFC13J6nJVPLlyU3nO
+
+Use your favourite editor to create ``~/bitwarden_rs/.env`` with the following content:
+
+.. code-block:: ini
+
+ ADMIN_TOKEN=PASTE YOUR TOKEN GENERATED ABOVE HERE # generate one with ~$ openssl rand -base64 48
+ ROCKET_PORT=62714 # your port here
+
+ SMTP_HOST=stardust.uberspace.de
+ SMTP_FROM=isabell@uber.space
+ SMTP_PORT=587
+ SMTP_SSL=true
+ SMTP_USERNAME=isabell@uber.space
+ SMTP_PASSWORD=MySuperSecretPassword
+ 
+ DOMAIN=https://isabell.uber.space
+
+Replace the mail placeholder variables with your valid IMAP credentials, otherwise the bitwarden_rs server will not be able to send you mail notifications or tokens to verify newly created user accounts.
+``SMTP_USERNAME`` and ``SMTP_PASSWORD`` must be the login data from a valid mail account. Replace the server domain with your final URL.
+
+.. note :: You can configure any type of service here, you're not limited to an uberspace IMAP user. If you prefer e.g. gmail refer to their documentations for ``SMTP_Port`` etc. accordingly.
+
+You can edit other options, look into .env.template to see a list of available options.
+
+
+Configuration
+=============
+
+Setup web backend
+-----------------
+
+.. note:: Enter the port you configured in the ``env`` file before.
+
+If you want to use a subdomain refer to the :manual:`web-backend manual <web-backends>`.
+
+.. include:: includes/web-backend.rst
+
+Setup web vault
+---------------
+
+Now it's time to test if everything works. If there is no error, you are good to go. You should be able to access your vault on ``https://isabell.uber.space``
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd ~/bitwarden_rs
+ [isabell@stardust bitwarden_rs]$ target/release/bitwarden_rs
+ /--------------------------------------------------------------------\
+ |                       Starting Bitwarden_RS                        |
+ |                      Version 1.14.1-843604c9                       |
+ |--------------------------------------------------------------------|
+ | This is an *unofficial* Bitwarden implementation, DO NOT use the   |
+ | official channels to report bugs/features, regardless of client.   |
+ | Report URL: https://github.com/dani-garcia/bitwarden_rs/issues/new |
+ \--------------------------------------------------------------------/
+
+ [2020-04-03 17:27:40][start][INFO] Rocket has launched from http://0.0.0.0:62714
+
+Setup daemon
+------------
+
+Use your favourite editor to create ``~/etc/services.d/bitwarden_rs.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:bitwarden_rs]
```

Comment:
Other guides do not indent the key/value pairs. While I fully understand the case for indenting them, i'd like to keep this consistent. Do you mind removing the tabs?

Reply:
Oh, thanks for the hint. Didn't notice that sweet detail. I removed the tabs and streamlined it :)